incubator/java-microprofile/README.md

```diff
@@ -44,6 +44,11 @@ OpenAPI endpoints:
 - http://localhost:9080/openapi (the RESTful APIs of the inventory service)
 - http://localhost:9080/openapi/ui (Swagger UI of the deployed APIs)
 
+### Quick start security
```

Comment:
This is good.. but did you look at the java-openliberty stack ?   Worth using the same doc?   (Not saying that one's better).